Heads of factions.sql

```diff
@@ -0,0 +1,24 @@
+-- Creature_equip_template
+
+DELETE FROM `creature_equip_template` WHERE `CreatureID` IN('14720', '107574');
+
+INSERT INTO `creature_equip_template` (`CreatureID`, `ID`, `ItemID1`, `ItemID2`, `ItemID3`, `VerifiedBuild`) VALUES
+(14720,1,21580,0,0,18019),
+(107574,1,12748,0,0,18019);
+
+ -- Creature
+
+DELETE FROM `creature` WHERE `guid` IN ('314277', '314300','314301','310903');
+DELETE FROM `creature` WHERE `id` IN ('1747', '45253'); 

```

Comment:
add a delete to 14720, 39605, 29611, 86832 and 107574, not necessary but it would be useful